.vscode/settings.json

```diff
@@ -9,5 +9,14 @@
     "python.testing.pytestArgs": ["tests"],
     "python.testing.pytestEnabled": true,
     "python.testing.unittestEnabled": false,
-    "rust-analyzer.linkedProjects": ["./src/secrets/Cargo.toml"],
+    "rust-analyzer.linkedProjects": [
+        "./src/secrets/Cargo.toml"
+    ],
+    "python.analysis.extraPaths": [
+        "./src/core",
+        "./src/zos_console",
+        "./src/zos_files",
+        "./src/zos_jobs",
+        "./src/zosmf"
```

Comment:
Thanks for adding this setting 🙂  Curious if it supports wildcards? If so it could be simplified to just one line: `./src/*`

Reply:
I don't think it supports wildcard as I tried it on VS Code.